numeric.c

```diff
@@ -1103,10 +1124,20 @@ rb_float_plus(VALUE x, VALUE y)
 }
 
 /*
- * call-seq:
- *   float - other  ->  float
+ *  call-seq:
+ *    self - other -> float
+ *
+ *  Returns a new \Float which is the difference of +self+ and +other+:
+ *
+ *    f = 3.14
+ *    f - 1                 # => 2.14
+ *    f - 1.0               # => 2.14
+ *    f - Rational(1, 1)    # => 2.14
+ *    f - Complex(1, 0)     # => (2.14+0i)
+ *    f - (Float::MAX * 2)  # => -Infinity
```

Comment:
Similarly, these last three examples can be dropped.

Reply:
Removed.